inc/namespace.php

```diff
@@ -59,9 +71,34 @@ function enqueue_scripts() : void {
 		$asset_file['version'],
 		false
 	);
+
+	$data = apply_filters( 'amf/script/data', [
+		'providers' => ProviderRegistry::instance()->get_script_data(),
```

Comment:
We could return `null` if the providers list is empty to make the JS logic simpler, otherwise we should cast this to an `(object)` for consistency